crates/uv-toolchain/src/managed.rs

```diff
@@ -22,15 +54,15 @@ pub struct InstalledToolchains {
 
 impl InstalledToolchains {
     /// A directory for installed toolchains at `root`.
-    pub fn from_path(root: impl Into<PathBuf>) -> Result<Self, io::Error> {
+    pub fn from_path(root: impl Into<PathBuf>) -> Result<Self, Error> {
```

Comment:
Nit: Should this be pub (will there be future usages where there are outside calls to `InstalledToolchains::from_path` directly)?